packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroupToolbarItem.tsx

```diff
@@ -1,31 +1,35 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/ChipGroup/chip-group';
+import { headingLevelType } from '../../helpers/heading';
 
 export interface ChipGroupToolbarItemProps extends React.HTMLProps<HTMLUListElement> {
   /**  Category name text */  
   categoryName?: string;
   /** Content rendered inside the chip text */
-  children: React.ReactNode; 
+  children: React.ReactNode;
   /** Additional classes added to the chip item */
-  className?: string;   
+  className?: string;
+  /** Set heading level to the chip item label */
+  headingLevel?: headingLevelType;
```

Comment:
Instead of `headingLevelType;` use `headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';`